lib/Game.rb

```diff
@@ -0,0 +1,70 @@
+
+class Game
+    include UserInterface
+    def initialize(board, player1, player2)
+        @board =  board
+        @player_1 = player1
+        @player_2 = player2
+        @current_player = @player_1
+    end
+    WIN_COMBO = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 4, 8],
+      [6, 4, 2]
+     ]
+
+    def play
+        loop do 
+            show_board(@board)
+            ask_move(@board , @current_player)
+            break if is_game_over?
+            @current_player = @current_player == @player_1 ? @player_2 : @player_1
+        end
+    end
+
+    def is_game_over?
+        if is_victory? == true
+            show_info(1)
+            return true
+        elsif(is_draw? == true)
+            show_info(0)
+            return true
+        end
+        return false
+    end
+
+    private def show_info(code)
```

Comment:
Why don't we just have one `private` statement and just put all methods intended to be private under that?

Reply:
Please put `show_info` under the `private` statement in Line 50. The `private` statement right beside the method definition is redundant.